classes/class-loader.php

```diff
@@ -33,50 +33,18 @@ final class Loader {
 	 * @var Loader $instance
 	 */
 	private static $instance;
-	/**
```

Comment:
All of these were really not needed. 2 just had a constant assigned to them and constant could be used instead. The others were just used locally in one place so there's no need to put them on an instance.